content/docs/jsx-in-depth.md

```diff
@@ -245,7 +245,31 @@ function App2() {
 }
 ```
 
-Spread attributes can be useful when you are building generic containers. However, they can also make your code messy by making it easy to pass a lot of irrelevant props to components that don't care about them. We recommend that you use this syntax sparingly.
+You can also pick specific props that your component will consume while passing all other props using the spread operator. 
+This ensures that the component consumes the `kind` prop only, and passes down all other props via `...other`.
+
+```js{2}
+const Button = props => {
+  const { kind, ...other } = props;
+  const className = kind === "primary" ? "PrimaryButton" : "SecondaryButton";
```

Comment:
> In response to: https://github.com/reactjs/reactjs.org/pull/121#discussion_r144336213 (comment thread was getting long)

Thanks for your patience, I think I finally understand what you've been trying to say 😭 
My motivation for re-adding this documentation was to surface how you can pass unknown props via spread attributes. But, I can see now that there are more considerations that need to be made when opting to use this syntax, which is what you were trying to communicate.  

Thanks again for being so thorough with this PR

Reply:
No problem at all 😄  It's easy to miscommunicate on a PR.